.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,16 @@
 <!-- DO NOT MODIFY OR DELETE THIS TEMPLATE. IT IS USED IN AUTOMATION. -->
-## Bug
 
-<!-- Search https://github.com/NuGet/Home/issues, and create one if you can't find a suitable issue. -->
-<!-- Paste the full link, like https://github.com/nuget/home/issues/1000. GitHub will render is neatly. -->
-Fixes:
+# Bug
 
-Regression? Last working version:
+<!-- If this is an engineering change or test change only, you do not need an issue. -->
+<!-- Find or create an issue in NuGet/Home and paste the full url. -->
+<!-- At the maintainers discretion, multiple changes may apply to a single issue, but only when the PRs are all created within a short period of time. -->
+Fixes:
 
 ## Description
-<!-- Add details about the fix. Include any information that would help the maintainer review this change effective. -->
 
 ## PR Checklist
 
-- [ ] PR has a meaningful title
-- [ ] PR has a linked issue.
-- [ ] Described changes
-
-- **Tests**
-  - [ ] Automated tests added
-  - **OR**
-  <!-- Describe why you haven't added automation. -->
-  - [ ] Test exception
-  - **OR**
-  - [ ] N/A <!-- Infrastructure, documentation etc. -->
-
-- **Documentation**
-  <!-- Please link the PR/issue if appropriate -->
-  - [ ] Documentation PR or issue filled
-  - **OR**
-  - [ ] N/A
+- [ ] Meaningful title, helpful description and a linked issue
+- [ ] Automated tests, a test exception request or does not apply
```

Comment:
What does the checkbox really mean? It would make it easier for me to review if it meant that the author _described_ the testing strategy.

Specifically, I see "automated tests updated" when new code is added, but this doesn't address validation strategy overall. Adding well-tested code does not mean the functional issue has been tested.

E.g.,
- Engineering? How did you or will you validate your change impacted CI reliability?
- UI? What UI tests were performed? Was Accessibility & PLOC testing completed per the guides?

```suggestion
- [ ] Validation strategy explained (Automated tests updated, UI & A11y testing completed (see results/screenshots), or CI analyzed or will be monitored), or does not apply
```

Reply:
An aspect of simplifying the template that I did not add was making the sentences shorter and simpler.

I'm literally trying to merge the 3 things we had previous into 1. We're just asserting tests were considered and added and give room for someone to add reasoning for an exception.

I think adding details about UI & A11Y testing is probably a bit more than we want. We do have the new feature doc which talks about UI considerations there and maybe that can be reworked, but in general I feel like adding too much specificity about the types of test in the template is something we should avoid.

Reply:
```suggestion
- [ ] Added tests
```

This, IMO, simplifies it and make it sufficiently clear. People who are just changing docs, or doing CI work, etc, they'll probably just comment themselves "this can't be tested", or whatever. But it makes it really clear that for product changes we want tests added.

Reply:
> I think adding details about UI & A11Y testing is probably a bit more than we want.

Ok, then omit my examples that are in parenthesis. I still think validation strategy makes more sense. I don't care whether tests are added or not, I want the bigger context. I can look at the code to see if tests are added.

Reply:
Can you clarify what you mean by bigger context? 

The idea behind the tests check is supposed to be a forcing function for people to justify when they don't add tests. 

If this is a larger feature, I feel like a validation strategy is more relevant, but feel like more often than not, that's many PRs, over a larger period of time, almost certainly with a design doc etc. 
Our goal is to target the majority case.

Reply:
Validation strategy is a bigger context than adding a test. What approach is being used? Describe that. It may not involve tests. It might involve some new tests or updating existing tests. 

Doesn't have to be a large feature. You can have a validation strategy for a 1 line change to fix a bug. "Installed .NET SDK 9.0 preview 7 and validated the new output by running the command". That's validation.

Reply:
Isn't that focusing on the minority scenario too much?
Unless it's a large feature, I largely rely on my automation. 
 
I think overall though, we want to make this light. We've had scenarios where tests weren't written, by both people internal and external, so this simple line allows them to "defend" why not.

Reply:
If you want to make it light, just drop this "added tests" checkbox entirely. This is easily identified by looking at the files changed, which we should all be reviewing tests anyway during the PR review.

Reply:
> so this simple line allows them to "defend" why not.

Have an example of this or know how often this happens? It's easy to explain why you didn't add tests. However, that doesn't mean you can't explain your validation strategy.

Reply:
The idea is that we simplify things to the bare necessities or things that may get frequently forgotten.
It's not uncommon for people not to add tests in at least 1 iteration of their PR. 
Example: